### DISCUSSION TITLE
希望增加 使用"//"方式的 注释(备注) 模式功能

### DISCUSSION THREAD

**Discussion by lirener** [NewFuture/DDNS#247] [Ideas]
希望 可以这样 参考 出一个以"//"为注释的 conf配置方式

```makefile
{
	//更多配置方式 参考：https://github.com/NewFuture/DDNS
	"$schema": "https://ddns.newfuture.cc/schema/v2.json", //配置模版？ 事例
	"debug": false,	//false=假(不开), true=真(启动)
	"dns": "dnspod_com",	//更新域名供应商："dnspod" 或 "dnspod_com" 或 "alidns" 或 "dnscom" 或 cloudflare"
	"id": "1008666",		// ID or Email
	"token": "ae86$cbbcctv666666666666666",	// API Token或Key
	"ipv4": ["test.lorzl.ml"],	//需要更新IPV4域名 可多个用,隔开
	"ipv6": ["test.lorzl.ml"],	//需要更新IPV6域名 可多个用,隔开 例子：["google.com","www.google.com"]
	"index4": "public",		//更新IPV4的方式 "public"=从外网(自定义接口)获取
	"index6": "default",	//更新IPV6的方式 "default"=默认 从 网卡 获取 
	"proxy": null	//代理方式 "127.0.0.1:1080;DIRECT" null/0=不代理 DIRECT=直连
}
```

**Comment by lirener** [comment]
<a href="https://ibb.co/cv9Bw61"><img src="https://i.ibb.co/JdT1qvK/QQ-20191102223653.jpg" alt="QQ-20191102223653" border="0" /></a>

**Comment by ericellis7** [comment]
哈咯，您好。我这里初步配置了一下，但是觉得还是用不起来，，，肯定是有些地方配置的不对。能麻烦您指导一下么？谢谢

**Comment by lirener** [comment]
你好 @EricMzky 
我是Windows系统的 我的配置如下 但是不同的就是没有加//

![EV~P863}PFMZAYYS68 X26M](https://user-images.githubusercontent.com/19354130/73612648-7859ab00-4628-11ea-9d50-ccf2aa54fc08.png)

![2222222222222](https://user-images.githubusercontent.com/19354130/73612668-9de6b480-4628-11ea-8e3c-0d705875ab62.jpg)

**Comment by d-w-x** [comment]
这里的话ddns可能可以考虑使用 json5 这个项目，它提供带注释的json。

**Comment by NewFuture** [comment]
最新测试版支持了
